packages/astro/src/cli/add/index.ts

```diff
@@ -83,20 +83,16 @@ export default async function seed() {
 `,
 	CLOUDFLARE_WRANGLER_CONFIG: (name: string) => `\
 {
-	"main": "dist/_worker.js/index.js",
+	"compatibility_date": "2025-05-21",
```

Comment:
We could export a util from `@cloudflare/vite-plugin` to get the latest `compatibility_date` supported by the installed version. You could then use that to provide the value here. Would that be helpful?

In the meantime, it would be good to use a more recent compatibility date.

Reply:
Ah yeah that probably does make sense.